src/plugins/external_tool_support/src/cufflinks/CufflinksWorker.cpp

```diff
@@ -365,6 +366,9 @@ Task *CufflinksWorker::tick() {
 
         return cufflinksSupportTask;
     } else if (input->isEnded()) {
+        if (output->hasMessage() == 0) {
```

Comment:
If the worker already processed some messages, then the output bus could contain messages.